website/src/pages/guides/successful-monolith-to-microservices-transition.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FunctionComponent } from 'react'
+import { PageProps } from 'gatsby'
+
+import Layout from '../../components/Layout'
+import { useCerosScript } from '../../hooks/ceros'
+
+const MonolithToMicroservices: FunctionComponent<PageProps> = props => {
+    useCerosScript()
+
+    return (
+        <Layout
+            location={props.location}
+            meta={{
+                title: '5 key elements of successful monolith-to-microservices migrations',
+                description:
+                    'Employ the five key techniques that the best engineering organizations enable to successfully conduct major architectural transitions.',
+            }}
+        >
+            <section
+                style={{
+                    position: 'relative',
+                    width: 'auto',
+                    padding: '0 0 286.02%',
+                    height: '0',
+                    top: '0',
+                    left: '0',
+                    bottom: '0',
+                    right: '0',
+                    margin: '0',
+                    border: '0 none',
+                }}
+                id="experience-627983184b4c8"
+                data-aspectratio="0.34963125"
+                data-mobile-aspectratio="0.14484881"
+            >
+                <iframe
+                    allowFullScreen
+                    src="https://view.ceros.com/sourcegraph/successful-monolith-to-microservices-transition?heightOverride=3661&mobileHeightOverride=5523"
+                    style={{
+                        position: 'absolute',
+                        top: '0',
+                        left: '0',
+                        bottom: '0',
+                        right: '0',
+                        margin: '0',
+                        padding: '0',
+                        border: '0 none',
+                        height: '1px',
+                        width: '1px',
+                        minHeight: '100%',
+                        minWidth: '100%',
+                    }}
+                    frameBorder="0"
+                    className="ceros-experience"
+                    title="5 key elements of successful monolith-to-microservices migrations"
+                    scrolling="no"
+                />
+            </section>
+        </Layout>
```

Comment:
I share your observations about responsiveness. The embedded text is difficult for users to read at mobile resolutions. Might we suggest developing a mobile variant as described here: [Ceros Adaptive Layouts](https://www.ceros.com/educate/help_center/adaptive-layouts-how-to-create-mobile-variants/)? 

Seeing as this is embedded content our options to support mobile-first are a bit limited. Similarly, I am hesitant to suggest a conditional notice, like "this page is best experienced on desktop". I want to be mindful of scope, so this may be more appropriate for a separate issue. What do you think? 

<img width="200" alt="Screen Shot 2022-05-17 at 1 55 53 PM" src="https://user-images.githubusercontent.com/60713139/168889229-a062674a-d591-41b4-9a75-a541d4f5dcb3.png">

Reply:
Yea so I learned Ceros is adaptive not responsive. If you change the screen size then refresh it'll usually adjust the spacing. [Slack thread for context](https://sourcegraph.slack.com/archives/C03FNHG83EZ/p1652814834461329?thread_ts=1652778716.435319&cid=C03FNHG83EZ). Although, I think there's still gaps with XL screens.

Reply:
Ah, it makes sense why I missed this with browser tools! Thank you for clarifying and for linking to this discussion. I was able to reproduce the size adaptation at a mobile resolution by refreshing. It looks good to me:

<img width="176" alt="Screen Shot 2022-05-19 at 8 38 39 AM" src="https://user-images.githubusercontent.com/60713139/169306850-d05887ab-0951-4c79-b2c7-f336f15fc912.png">